ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -1125,39 +1126,59 @@ impl TraceBuilder {
                     args: x.args.clone(),
                 })
                 .collect::<Vec<_>>();
+
+            //#[cfg(tracer_swt)]
```

Comment:
This commented code looks fishy.

Reply:
Whoops. That definitely needs to go back in. I commented it out to get rust-analyser to give me hints.

Reply:
[b5a496d](https://github.com/ykjit/yk/pull/1422/commits/b5a496d546cf2f1eee21b143b60bf77ece9bbb59)